### PR TITLE
add option to specify save strategy

### PR DESCRIPTION
**Pull request by veriff-saulvargas**
Add the ability to specify save strategy.

### CODE REVIEW

**Review by frostming — APPROVED** [review]

**Comment by veriff-saulvargas** [comment]
@frostming I fixed the conflicts with the other PR, I think it is ready for merging.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ with:
   pr-title: "Update dependencies"
   # The update strategy, can be 'reuse', 'eager' or 'all'
   update-strategy: eager
+  # The save strategy, can 'compatible', 'wildcard', 'exact' or 'minimum'
+  save-strategy: compatible
```
Comment (frostming) [inline README.md]:
```suggestion
  save-strategy: minimum
```
Correct the default value

Comment (veriff-saulvargas) [reply README.md]: Ok, will do, but how about `update-strategy` and `unconstrained`, should they also have the default values, i.e. `reuse` and `false, respectively?

Comment (frostming) [reply README.md]: Oh sure, i didn't notice it is the example in readme

**Pull request merged**